council/council.html

```diff
@@ -146,6 +146,44 @@ <h3 id="team-fo"><a href="#team-fo">Team attempts to resolve Formal Objections</
     <li><p>If the resolution causes a change in a proposed charter, then all participants in the AC review of that charter must have an opportunity to review the revised proposed charter.</p></li>
   </ul>
 
+  <h3 id="publishing-fos"><a href="#publishing-fos">Publishing formal objections</a></h2>
+
+  <p>
+    Formal objections must be made public using <a
+    href='https://lists.w3.org/Archives/Public/public-review-comments/'>public-review-comments</a> and no later than when the
+    relevant council is initiated. The team should publish formal objections as soon as
+    reasonable within the 90 days investigation and mediation period, with some
+    considerations including:
+  </p>
+  <ul>
+    <li>all Formal Objections must be published, even those that are expected to
+      be / have been withdrawn.
+    </li>
+    <li>The Formal Objection ought to be made public as soon as it is
```

Comment:
to match capitalization of the rest of the list
```suggestion
    <li>the Formal Objection ought to be made public as soon as it is
```